### PR TITLE
Update admin-submission.md

### PR DESCRIPTION
**Pull request by andypunt**
Updated admin submissions flow.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 1]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
 - **Submission type**
 - **Delivery reason**
 - **Status**<sup>\*</sup>
-- **Control type**
-- **Control source**
 
   <sup>\*</sup> If you click this value, detailed information is displayed in a flyout.
 
+#### Admin submission rescan details
+Messages that are submitted in Admin submissions are rescanned and results shown in the details flyout:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
Messages that are submitted in admin submissions are rescanned and results shown in the details flyout:
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 2]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
...
+1) if there was a failure in the sender's email authentication at time of delivery
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
1) If there was a failure in the sender's email authentication at the time of delivery.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 3]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
...
+2) Information about any policy hits that could have affected or overridden the verdict of a message
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
2) Information about any policy hits that could have affected or overridden the verdict of a message.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 4]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
...
+3) current detonation results to see if the URLs or files contained in the message were malicious or not
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
3) Current detonation results to see if the URLs or files contained in the message were malicious or not.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 5]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
...
+4) feedback from graders
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
4) Feedback from graders.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 6]
```diff
@@ -140,11 +136,18 @@ You can click the **Column options** button near the bottom of the page to add o
...
+
+If a override was found, the rescan should complete in several minutes. If there wasn't a problem in email authentication or delivery wasn't affected by an override, then the feedback from graders could take up to a day.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
If an override was found, the rescan should complete in several minutes. If there wasn't a problem in email authentication or delivery wasn't affected by an override, then the feedback from graders could take up to a day.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**